packages/components/src/VirtualizedList/CellTitle/CellTitleActions.component.js

```diff
@@ -31,16 +63,21 @@ function CellTitleActions({ rowData, actionsKey, displayMode, persistentActionsK
 	);
 }
 
-CellTitleActions.displayName = 'VirtualizedList(CellTitleActions)';
-CellTitleActions.propTypes = {
+CellTitleActionsComponent.displayName = 'VirtualizedList(CellTitleActions)';
+CellTitleActionsComponent.propTypes = {
+	id: PropTypes.string,
 	// The actions property key. Actions = props.rowData[props.actionsKey]
 	actionsKey: PropTypes.string,
 	// The persistent actions property key. Actions = props.rowData[props.persistentActionsKey]
 	persistentActionsKey: PropTypes.string,
 	/** The display mode. */
 	displayMode: PropTypes.oneOf([TITLE_MODE_TEXT, TITLE_MODE_INPUT]),
 	// The collection item.
-	rowData: PropTypes.object, // eslint-disable-line
+	rowData: PropTypes.object,
+	t: PropTypes.func.isRequired,
+};
+CellTitleActionsComponent.defaultProps = {
+	t: getDefaultT(),
 };
 
-export default CellTitleActions;
+export default translate(I18N_DOMAIN_COMPONENTS, { i18n: DEFAULT_I18N })(CellTitleActionsComponent);
```

Comment:
Now, define a i18n instance to the HOC translateis not necessary. Just import import '../../translate' is useful to set a default i18n instance to react-i18next.
you can do translate(I18N_DOMAIN_COMPONENTS)(CellTitleActionsComponent);

Reply:
👍 this PR was months before your change, guess I have to adapt it :D